11-information_governance.Rmd

```diff
@@ -28,187 +28,119 @@ If requesting access to a data source, you should provide the GitHub usernames o
 * Files stored in users' home directories are backed up to Amazon S3 automatically.
 * Previous versions of files stored in users' home directories are also backed up to Amazon S3.
 * Once files are deleted from a user's home directory, the backup is retained for a further 90 days and can be restored.
-* To request that a file be restored or to access a previous version of a file, contact the [Analytical Platform team](mailto:analytical_platform@digital.justice.gov.uk).
+* To request that a file be restored or to access a previous version of a file, contact the Analytical Platform team at [analytical_platform@digital.justice.gov.uk](mailto:analytical_platform@digital.justice.gov.uk).
 
-#### Best practice guidelines
+## Data movements
 
-All users should comply with the following best practice guidelines for retaining personal data.
+You can move data classified as OFFICIAL and OFFICIAL-SENSITIVE onto the Analytical Platform. SECRET and TOP SECRET data is not allowed on the Analytical Platform.
 
-__All users will:__
+All data movements should take place safely and securely to ensure that data is protected at all times, including when in transit.
 
-* be aware when they are working with personal data;
-* consider and justify why personal data needs to be retained and for how long;
-* identify when personal data needs to be kept for public interest archiving, scientific or historical research, or statistical purposes;
-* review the need to retain personal data on a regular basis;
-* erase or anonymise personal data when it is no longer required; and
-* erase data as required in accordance with an individual's 'right to be forgotten' under the Data Protection Act 2018.
+The process of moving data to the Analytical Platform is different depending on whether you are working with personal data.
```

Comment:
I think 'will depend on' rather than 'is different depending on'

Reply:
I've reworked this in 2aa8cc2. Let me know if you think it works better.